SEED-platform/seed
# Fix and cleanup suborganizations

#### Any background context you want to provide?
There were several issues related to sub organizations. See the relevant tickets below.

#### What's this PR do?
* Add helper method for creating suborganizations
* Fix column list setting view set
* Return error when creating suborg and user does not exist

#### How should this be manually tested?
Profile List Settings (#1686)

* Create a suborg and create a member and a viewer
* As the member of the suborg, add data to the inventory
* Create a list profile setting as a suborg member
* Log out and login as a viewer of the suborg. 
* Verify list settings profile appear in inventory list
* Verify that the viewer user is unable to edit the list settings (need to report this back to the user)

Create Suborg (#1681)
* Create a suborg with a non existent user
* The dialog should nicely report that the user does not exist

Add data as suborg member (#1682)
* Create new suborg and add yourself as a member (or owner) of suborg
* Log into suborg
* Create new dataset
* Upload file and go to mapping screen
* The mapping should go through without any issues

#### What are the relevant tickets?
#1686 #1681 #1682
#### Screenshots (if appropriate)
#### Definition of Done:
